### DISCUSSION TITLE
1  device or multiple devices.

### DISCUSSION THREAD

**Discussion by Roeland54** [marq24/ha-evcc#6] [Ideas]
The current setup of the integration creates only one device with all entities inside.
Wouldn't it be more intuitive to have a device for each loadpoint and one device for the more general evcc data?

**Comment by marq24** [comment]
Thanks for your input - This would IMHO imply to pull data multiple times (per configured device) from the evcc backend, creating quite some extra load... Beside the fact, that IMHO the configuration/setup will be quite complicated - I have to check, if it's possible to "split" one config_entry into multiple devices... 

You have to full control grouping the entities you prefer in your local dashboards - right? But I might just have missed (did not understood yet), why you prefer to have multiple devices? I just have multiple devices here in my HA, if I have identical systems with different IP's (like my CAMs - or I am aware of a user of my Watterkotte integration where there is the need to control multiple heating systems in a single HA instance).

**Comment by UDicke** [comment]
I think that everyone can create dashboards / cards in HA to his own needs. See this example:
![grafik](https://github.com/marq24/ha-evcc/assets/161384698/d294e03b-2327-4cf0-902f-45f5518d7d71)
All Entities are available and I would prefer a solution like it is today to avoid  `This would IMHO imply to pull data multiple times (per configured device) from the evcc backend, creating quite some extra load... `

**Comment by Roeland54** [comment]
I am aware I can do what I want with entities in dashboards. It is just seamed more logical to have the entities separated in devices. Just like the philips hue intregration does have a separated device for each physical device. Now I have 1 device with 200 entities its is just hard to find the entity I need. 
It was just an idea. 

This integration is already way better than what we needed to do before. thanks for your effort.

**Reply by UDicke** [reply]
Ok, even just what I thougt while reading...

> This integration is already way better than what we needed to do before. thanks for your effort.

Yes, I agree, thank you Matthias!
